server/serverQueryManager.py

```diff
@@ -21,66 +22,64 @@ def add(self, args: dict) -> dict:
         datetime_obj = string_to_datetime(args['date'])
         results = []
         uids_on_day = self.schedule.get(datetime_obj)
-        response_code = 3
+        response_code = ResponseCode.FORBIDDEN
 
         for uid in uids_on_day:
-            if self.adjmat.is_adjacent(args['uid'], uid) == True:
+            if self.adjmat.is_adjacent(args['uid'], uid):
                 results.append(uid)
 
-
         if len(results) > 0:
             # uid can't work on the same day as someone already working on that day
-            response_code = 409
+            response_code = ResponseCode.CONFLICT
         else:
             added = self.schedule.add(args['uid'], datetime_obj)
             if not added:
                 # uid not added successfully
-                response_code = 417
+                response_code = ResponseCode.UNEXPECTED
             else:
                 # uid added to day successfully
-                response_code = 200
+                response_code = ResponseCode.OK
 
         response = {'responseCode': response_code, 'results': results}
         return response
 
     def remove(self, args: dict) -> dict:
         datetime_obj = string_to_datetime(args['date'])
+        response_code = ResponseCode.FORBIDDEN
         results = []
         uids_on_day = self.schedule.get(datetime_obj)
 
```

Comment:
We need to check if the uid is stored somewhere in our list, and warn them if they are not. (Maybe this should be a function that both add and remove call? userexists()?)